examples/src/test/java/com/opengamma/strata/examples/marketdata/ExampleMarketDataBuilderTest.java

```diff
@@ -242,14 +243,14 @@ private void assertBuilder(ExampleMarketDataBuilder builder) {
     assertEquals(MARKET_DATA_DATE, snapshot.getValuationDate().getSingleValue());
 
     for (ObservableId id : TIME_SERIES) {
-      assertTrue(snapshot.containsTimeSeries(id), "Time-series not found: " + id);
+      assertFalse(snapshot.getTimeSeries(id).isEmpty(), "Time-series not found: " + id);
     }
     assertEquals(snapshot.getTimeSeries().size(), TIME_SERIES.size(),
         Messages.format("Snapshot contained unexpected time-series: {}",
             Sets.difference(snapshot.getTimeSeries().keySet(), TIME_SERIES)));
 
     for (MarketDataId<?> id : VALUES) {
-      assertTrue(snapshot.containsValue(id), "Id not found: " + id);
+      assertTrue(snapshot.findValue(id).isPresent(), "Id not found: " + id);
```

Comment:
When testing reference data I got a speed up of 15% or so by directly implementing `getValue` not going via `findValue`. Thus, there is a performance question to consider.